tests/crypto/tests.c

```diff
@@ -9,6 +9,9 @@ void TestAll()
     TestASN1();
 #endif
     TestCRL();
+#if defined(OE_BUILD_ENCLAVE)
```

Comment:
Why is this ifdef needed here but not in tests.h?

Reply:
Updated